common/src/main/java/earth/terrarium/olympus/client/components/dropdown/DropdownEntry.java

```diff
@@ -31,7 +33,7 @@ protected void renderWidget(GuiGraphics graphics, int mouseX, int mouseY, float
 
         Component text = this.dropdown.getText(this.value);
         int textOffset = (this.height - 8) / 2;
-        graphics.drawString(Minecraft.getInstance().font, text, this.getX() + textOffset, this.getY() + textOffset, Dropdown.COLOR);
+        graphics.drawString(Minecraft.getInstance().font, text, this.getX() + textOffset, this.getY() + textOffset, COLOR);
```

Comment:
also allow the color and textures to be overrode here too to be consistent.

Reply:
its about principles not about how easy it is, if you wanted to just change the texture you have to copy everything in this render widget and just change the 2 textures